build.gradle

```diff
@@ -1,3 +1,26 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2016 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
```

Comment:
I have added it to main/top-level build.gradle file. This fixes the problem, but then another issue occurred.

The Jenkins server seems to have problems with fetching the FlowText-library from the given HTTPS-url. It says that the peer is not authenticated. I think, that this might be a TLS-related problem, since when opening the URL in my browser (or manually download the xml-file via curl/wget) I do not encounter any problems: https://jitpack.io/com/github/deano2390/FlowTextView/2.0.4/FlowTextView-2.0.4.pom

`FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':catroid'.
> Could not resolve all dependencies for configuration ':catroid:_standaloneDebugCompile'.
   > Could not resolve com.github.deano2390:FlowTextView:2.0.4.
     Required by:
         CatroidTriggerPullRequestTest:catroid:unspecified
      > Could not resolve com.github.deano2390:FlowTextView:2.0.4.
         > Could not get resource 'https://jitpack.io/com/github/deano2390/FlowTextView/2.0.4/FlowTextView-2.0.4.pom'.
            > Could not GET 'https://jitpack.io/com/github/deano2390/FlowTextView/2.0.4/FlowTextView-2.0.4.pom'.
               > peer not authenticated
      > Could not resolve com.github.deano2390:FlowTextView:2.0.4.
         > Could not get resource 'https://jitpack.io/com/github/deano2390/FlowTextView/2.0.4/FlowTextView-2.0.4.pom'.
            > Could not GET 'https://jitpack.io/com/github/deano2390/FlowTextView/2.0.4/FlowTextView-2.0.4.pom'.
               > peer not authenticated
`